extension/parquet/include/parquet_reader.hpp

```diff
@@ -71,10 +71,12 @@ struct ParquetColumnDefinition {
 	static ParquetColumnDefinition FromSchemaValue(ClientContext &context, const Value &column_value);
 
 public:
+	// DEPRECATED, use 'identifier' instead
```

Comment:
Do we actually need the identifier field here? Shouldn't the combination of `field_id` and `name` already provide enough information here?